msal/src/main/java/com/microsoft/identity/client/internal/controllers/LocalMSALController.java

```diff
@@ -135,11 +138,37 @@ private AuthorizationRequest getAuthorizationRequest(final OAuth2Strategy strate
             // Add additional fields to the AuthorizationRequest.Builder to support interactive
             request.setLoginHint(
                     acquireTokenOperationParameters.getLoginHint()
-            ).setExtraQueryParam(
+            ).setExtraQueryParams(
                     acquireTokenOperationParameters.getExtraQueryStringParameters()
             ).setPrompt(
                     acquireTokenOperationParameters.getUIBehavior().toString()
             );
+
+            // Set fields for session continuation, if applicable
```

Comment:
@shoatman @addev-ashish This change enables session continuation on the `MicrosoftStsAuthorizationRequest`

Reply:
You can move this logic into the strategy itself... into the method that returns the builder.... so that it doesn't leak into the controller.

Reply:
@shoatman Sounds great -- I've taken this suggestion and implemented it in [f73fb3b94318a6ca6a8ec9030c8ee4ea44ad1e08](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/232/commits/f73fb3b94318a6ca6a8ec9030c8ee4ea44ad1e08) --  this is available to view in https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/232